.gitignore

```
@@ -328,3 +328,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+Mongo/Mongo.Migrations/Defaults.cs
+Api.FacePlusPlus/Defaults.cs
+Api.Flickr/Defaults.cs
```

Can be changed to single line
```suggestion
Defaults.cs
```